index.json

```diff
@@ -1,6 +1,12 @@
 {
   "plugin_manager_url": "https://github.com/bombsquad-community/plugin-manager/{content_type}/{tag}/plugin_manager.py",
   "versions": {
+    "0.2.3": {
```

Comment:
Change this to 0.3.0 and we can merge,other stuff can be changed later

Reply:
Also remove the data and add value as null

Reply:
null in the sense remove everything in the braces "{}"?

Reply:
Yes

Reply:
Check the readme, it's just like updating plugin